spec/tester_spec.rb

```diff
@@ -1,4 +1,7 @@
-require 'spec_helper'
+# coding: utf-8
```

Comment:
Is this necessary?

Reply:
It's not. Will remove in a subsequent PR.